doc/sphinx/Makefile

```diff
@@ -35,12 +36,12 @@ doxy-code: gen_python_api_doc
 html: doxy-code
 
 pickle:
-	$(SPHINXBUILD) -b pickle $(ALLSPHINXOPTS) $(BUILDDIR)/pickle
+	$(SPHINXBUILD) -b pickle $(SOURCEDIR) $(BUILDDIR)/pickle
```

Comment:
Recommend setting ALLSPHINXOPTS = ${SOURCEDIR} around line 10 instead of these duplicate changes in the individual target lines

Reply:
I tried this and it breaks doxygen build for some reason.  We don't use most of the other targets, they are just there for maybe future purposes.